CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [1.2.16](https://github.com/ably/ably-js/tree/1.2.16) (2022-01-14)
+- Fix bug where channel rewind would ignore messages after reattaching
```

Comment:
Is there a reason you've not linked to the PR (or issue) here?

Reply:
My mistake, ca4da31